site/src/sphinx/advanced-structured-logging.rst

```diff
@@ -61,6 +61,31 @@ What properties can be retrieved?
 | ``responseContentPreview``  | the preview of the response content                                  |
 +-----------------------------+----------------------------------------------------------------------+
 
++--------------------------------------------------------------------------------------------------------------+
+| Connection properties (Client side only)                                                                     |
++==========================================+===================================================================+
+| ``dnsResolutionStartTimeMicros``         | when resolving a domain name started                              |
++------------------------------------------+-------------------------------------------------------------------+
+| ``dnsResolutionDurationNanos``           | the duration took to resolve a domain name                        |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectStartTimeMicros``         | when connecting to a remote peer started                          |
++------------------------------------------+-------------------------------------------------------------------+
+| ``socketConnectDurationNanos``           | the duration took to connect to a remote peer                     |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionStartTimeMicros``    | when waiting the completion of an ongoing connecting attempt      |
+|                                          | started                                                           |
++------------------------------------------+-------------------------------------------------------------------+
+| ``pendingAcquisitionDurationNanos``      | the duration took to wait the completion of an ongoing connecting |
+|                                          | attempt                                                           |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionStartTimeMicros`` | when acquiring a connection started                               |
++------------------------------------------+-------------------------------------------------------------------+
+| ``connectionAcquisitionDurationNanos``   | the duration took to get a connection. This value is greater than |
+|                                          | or equal to the sum of ``dnsResolutionDurationNanos``,            |
+|                                          | ``socketConnectDurationNanos`` and                                |
+|                                          | ``pendingAcquisitionDurationNanos``                               |
++------------------------------------------+-------------------------------------------------------------------+
+
```

Comment:
How about adding an ASCII diagram that shows the typical scenarios:

```
|---------- connectionAcquisition ----------|
|--- dnsResolution ---|
                      |--- socketConnect ---|
```

and

```
|------------- connectionAcquisition ------------|
|--- dnsResolution ---|
                      |--- pendingAcquisition ---|
```

I guess we will have to specify that these are not the only possible scenarios.